.github/workflows/ci.yml

```diff
@@ -303,6 +305,8 @@ jobs:
         PYTHON:
           - {VERSION: "3.7", NOXSESSION: "tests-nocoverage"}
           - {VERSION: "3.13", NOXSESSION: "tests"}
+          - {VERSION: "pypy-3.10", NOXSESSION: "tests-nocoverage"}
+          - {VERSION: "pypy-3.11", NOXSESSION: "tests-nocoverage"}
```

Comment:
It's still here.

Reply:
Oohh, sorry, I totally misunderstood you. I thought you were only talking about the chunk which prevented the CI from testing on winx86.

Fixed!